semantic_release/history/logs.py

```diff
@@ -19,7 +19,7 @@
 
 # Sections which will be shown in the Markdown changelog.
 # This is NOT related to supported commit types.
-CHANGELOG_SECTIONS = [
+DEFAULT_CHANGELOG_SECTIONS = [
```

Comment:
Why not use `defaults.cfg`?

Reply:
Ah, I'm just not familiar with the code, will change.